c8/c8ql.py

```diff
@@ -337,3 +337,20 @@ def response_handler(resp):
             return True
 
         return self._execute(request, response_handler)
+
+    def export_data_query(self, query, bind_vars):
+        data = {'query': query}
```

Comment:
Hello Edgar, I guess we should add the information( description) about the function here too?